modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/helper/WorkflowMetricsRESTTestHelper.java

```diff
@@ -804,16 +828,69 @@ private Document _creatWorkflowMetricsSLAInstanceResultDocument(
 		).setValue(
 			"processId", processId
 		).setValue(
-			"slaDefinitionId", slaDefinitionId
+			"remainingTime", remainingTime
 		).setValue(
-			"status", "RUNNING"
-		).setString(
+			"slaDefinitionId", slaDefinitionId
+		);
+
+		if (status != null) {
+			documentBuilder.setValue("status", status.getValue());
+		}
+		else {
+			documentBuilder.setValue(
+				"status", SLAResult.Status.RUNNING.getValue());
+		}
+
+		documentBuilder.setString(
 			"uid",
 			_digest(
 				"WorkflowMetricsSLAInstanceResult", companyId, instanceId,
-				processId, slaDefinitionId)
+				processId, slaDefinitionId));
+
+		return documentBuilder.build();
+	}
+
+	private Document _creatWorkflowMetricsSLAInstanceResultDocument(
```

Comment:
Hey @MarinhoFeliphe, Can we adapt the code to have only one `_creatWorkflowMetricsSLAInstanceResultDocument`?